arrow-arith/src/temporal.rs

```diff
@@ -89,7 +89,7 @@ where
     T: ChronoDateExt + Datelike + Timelike,
 {
     match part {
-        DatePart::Quarter => |d| d.quarter() as i32,
+        DatePart::Quarter => |d| <dyn ChronoDateExt>::quarter(&d) as i32,
```

Comment:
delta-rs complains about ambiguity here between ChronoDateExt and DateLike. I guess more recent versions both have quarter. Not sure what to do here.

Reply:
I'm not sure we can pack this change into a decryption PR.

Reply:
We could perhaps argue for it to be a separate one line PR?

Reply:
Just skip this for now. I'm hoping it just turns out to be some package mismatch upstrwam